site/src/modules/management/ManagementSettingsLayout.tsx

```diff
@@ -48,61 +47,134 @@ export const canEditOrganization = (
 	);
 };
 
+export const isManagementRoutePermitted = (
```

Comment:
yeah, this feels like total overkill to me, especially for a quick patch this is a _huge_ departure from how we handle this sort of thing everywhere else in the app